account_invoice_production_lot/report/report_invoice.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="account_invoice_production_lot_report" inherit_id="account.report_invoice_document">
+
+        <xpath expr="//span[@t-field='line.name']" position="after">
+            <t groups="stock.group_production_lot">
+                <t t-set="lots" t-value="line.lots_grouped_by_quantity().items()"/>
+                <t t-if="len(lots) == 1">
+                     S/N: <span t-esc="list(lots)[0][0]"/> (<span t-esc="list(lots)[0][1]"/>)
```

Comment:
This would have them all (no need to put the qty as they are unique :)
```suggestion
                     S/N: <span t-esc="', '.join([x[0] for x in lots])"/>
```